### DISCUSSION TITLE
Support syntax highlight in Issue view

### DISCUSSION THREAD

**Discussion by snpefk** [ankitpokhrel/jira-cli#550] [General]
I noticed that issue viewer does not highlight code after I specify into code block used language

**Comment by ankitpokhrel** [comment]
Hi @snpefk, could you provide more details please? For example this is how the following codeblock is rendered in my setup (Jira Server 8.22.1).

```
{code:go}
package main

import "fmt"

func main() {
	fmt.Println("Hello, 世界")
}{code}
```

![Screen Shot 2023-01-13 at 6 56 58 PM](https://user-images.githubusercontent.com/2364546/212388219-d773683b-76ae-424a-99cc-6cea5f11d634.png)

**Reply by snpefk** [reply]
Yeah, my bad. I double checked and syntax highlight was on spot. Must be some misconfiguration or just inattention.

However, while I was rechecking issues I noticed that single line code block like `{{some code}}` prints as is. For example: 
![Screenshot from 2023-03-22 19-10-27](https://user-images.githubusercontent.com/2883201/226950166-7ad8be56-4f45-42cf-9768-5daaaf116bb2.jpg)
![Screenshot 2023-03-22 at 19-10-02 CORE-4276 Тестовая задача - Cardsmobile Jira](https://user-images.githubusercontent.com/2883201/226950177-23ac2448-04d0-4b69-8b4a-f935287e6ab7.png)
